drlineconfiguration.json

```diff
@@ -946,17 +1024,33 @@
     "name": "ZFIN",
     "displayName": "ZFIN",
     "category": "ORG",
-    "uriLink": "https://zfin.org/%s",
+    "uriLink": "https://zfin.org/%id",
     "attributes": [{
       "name": "GeneName",
       "xmlTag": "gene designation"
     }]
   },
+  {
+    "name": "HUGE",
+    "displayName": "HUGE",
+    "category": "ORG",
+    "linkTp": "Implicit",
+    "linkedReason": "GN:KIAAnnnn, Human Entries, can be multiple",
+    "uriLink": "http://www.kazusa.or.jp/huge/gfpage/%gene"
+  },
+  {
+    "name": "Rouge",
+    "displayName": "Rouge",
+    "category": "ORG",
+    "linkTp": "Implicit",
+    "linkedReason": "GN:KIAAnnnn, Mouse Entries, can be multiple",
+    "uriLink": "http://www.kazusa.or.jp/rouge/gfpage/%gene"
+  },
   {
     "name": "eggNOG",
     "displayName": "eggNOG",
     "category": "PLG",
-    "uriLink": "http://eggnogdb.embl.de/#/app/results?seqid=%u&target_nogs=%s",
+    "uriLink": "http://eggnogdb.embl.de/#/app/results?seqid=%u&target_nogs=%id",

```

Comment:
`%u` here should be `accession` eg: `http://eggnogdb.embl.de/#/app/results?seqid=P05067&target_nogs=KOG3540`